src/algebra/group_action_hom.lean

```diff
@@ -0,0 +1,259 @@
+/-
+Copyright (c) 2020 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau
+-/
+import algebra.group_ring_action
+
+/-!
+# Equivariant homomorphisms
+
+## Main definitions
+
+* `mul_action_hom M X Y`, the type of equivariant functions from `X` to `Y`.
```

Comment:
We don't know what structures M, X and Y are meant to have here. It makes this whole docstring unusable.